Makefile

```diff
@@ -118,9 +118,9 @@ ifndef $(GOPATH)
 	export GOPATH
 endif
 
-ARGOEXEC_PKGS    := $(shell echo cmd/argoexec            && go list -f '{{ join .Deps "\n" }}' ./cmd/argoexec/            | grep 'argoproj/argo-workflows/v3/' | cut -c 39-)
-CLI_PKGS         := $(shell echo cmd/argo                && go list -f '{{ join .Deps "\n" }}' ./cmd/argo/                | grep 'argoproj/argo-workflows/v3/' | cut -c 39-)
-CONTROLLER_PKGS  := $(shell echo cmd/workflow-controller && go list -f '{{ join .Deps "\n" }}' ./cmd/workflow-controller/ | grep 'argoproj/argo-workflows/v3/' | cut -c 39-)
+ARGOEXEC_PKG_FILES    := $(shell echo cmd/argoexec            && go list -f '{{ join .Deps "\n" }}' ./cmd/argoexec/            | grep 'argoproj/argo-workflows/v3/' | cut -c 39- | xargs -I {} find {} -type f -name "*.go")
+CLI_PKG_FILES         := $(shell echo cmd/argo                && go list -f '{{ join .Deps "\n" }}' ./cmd/argo/                | grep 'argoproj/argo-workflows/v3/' | cut -c 39- | xargs -I {} find {} -type f -name "*.go")
+CONTROLLER_PKG_FILES  := $(shell echo cmd/workflow-controller && go list -f '{{ join .Deps "\n" }}' ./cmd/workflow-controller/ | grep 'argoproj/argo-workflows/v3/' | cut -c 39- | xargs -I {} find {} -type f -name "*.go")
```

Comment:
Can you paste the outputs of these? We need to confirm this change is correct

Reply:
Here we go. I hope you enjoy! 
[cli_pkg_files.txt](https://github.com/user-attachments/files/15737780/cli_pkg_files.txt)
[controller_pkg_files.txt](https://github.com/user-attachments/files/15737781/controller_pkg_files.txt)
[argoexec_pkg_files.txt](https://github.com/user-attachments/files/15737783/argoexec_pkg_files.txt)

Reply:
@terrytangyuan please could you take another look here, you're blocking this one.